src/configs/binary-elementwise-config.c

```diff
@@ -523,6 +528,67 @@ static void init_f32_vcopysign_config(void) {
   #endif
 }
 
+
+static void init_s32_vmul_config(void) {
+  #if XNN_ARCH_ARM
+    const struct xnn_hardware_config* hardware_config = xnn_init_hardware_config();
+    assert(hardware_config != NULL);
+    if (hardware_config->use_arm_neon){
+      s32_vmul_config.minmax.op_ukernel = (xnn_vbinary_ukernel_fn) xnn_s32_vmul_minmax_ukernel__neon_u8;
```

Comment:
Please benchmark (where possible: x86 & arm) and choose the fastest

Reply:
will change this later with benchmark pr